src/app/header/header.component.html

```diff
@@ -37,20 +37,18 @@
           <i class="fas fa-bars"></i> <span class="caret"></span>
         </button>
         <ul *dropdownMenu class="dropdown-menu dropdown-menu-right" role="menu">
-          <li role="menuitem"><a class="dropdown-item" routerLink="/home" routerLinkActive='active'
-              [routerLinkActiveOptions]='{exact: true}'>Home</a></li>
+          <li role="menuitem">
+            <a class="dropdown-item" routerLink="/home" routerLinkActive='active'>Home</a>
+          </li>
           <li role="menuitem"><a class="dropdown-item" routerLink="/skills" routerLinkActive='active'>Skills</a></li>
```

Comment:
@Yuriy-27  again I would recommend to use *ngFor, only routerLink and text are different